fenix/app/src/main/java/org/mozilla/fenix/home/topsites/TopSiteItemViewHolder.kt

```diff
@@ -53,49 +53,6 @@ class TopSiteItemViewHolder(
     private val binding = TopSiteItemBinding.bind(view)
 
     init {
-        itemView.setOnLongClickListener {
```

Comment:
There is a small advantage is cleaning up the code but this change is not needed to solve the current issue.
You could add this refactoring in a separate commit but I would favor not modifying the code to keep the git history clean and ease the blame process later.